apps/bors_github/lib/github/server.ex

```diff
@@ -390,6 +390,31 @@ defmodule BorsNG.GitHub.Server do
   end
 
   @doc """
+  Uses a token from the cache, or, if the request fails,
+  retry without using the cached token.
+  """
+  @spec use_token!(ttoken, ttokenreg, ((ttoken) -> term)) ::
+    {:reply, term, ttokenreg}
```

Comment:
Yeah, it's used by dialyxir. That's one reason for having it; bors gates on dialyzer raising no warnings.